app/code/core/Mage/Core/Helper/Url.php

```diff
@@ -180,7 +180,7 @@ public function encodePunycode($url)
         $parsedUrl = parse_url($url);
         if (!$this->_isPunycode($parsedUrl['host'])) {
             if (function_exists('idn_to_ascii')) {
-                $host = idn_to_ascii($parsedUrl['host']);
+                $host = idn_to_ascii($parsedUrl['host'], IDNA_DEFAULT, INTL_IDNA_VARIANT_UTS46);
```

Comment:
the parameters are already set as default. why must this be specified explicitly now?

Reply:
I have added these default parameters some months ago, at this time, there was a notice or a warning with php 7.2? (I don't remember exactly). I tested again with php 7.2 & 7.4, and I'm agree these default parameters are useless. Sorry.